### PR TITLE
Add McDonald's as easy

### PR DESCRIPTION
**Pull request by WebSnke**
badabababa

### CODE REVIEW

**Comment by WebSnke** [comment]
Hi, can I add any kind of account to this list in the future?

**Review by tupaschoal — COMMENTED** [review]
What do you mean, to the same branch?

**Comment by WebSnke** [comment]
> What do you mean, to the same branch?

No, just in general.

**Comment by tupaschoal** [comment]
As long as it's not criminal

File: _data/sites.json [hunk 1]
```diff
@@ -9240,6 +9240,15 @@
             "home.mcafee.com"
         ]
     },
+    
+    {
+        "name": "McDonald's",
```
Comment (tupaschoal) [inline _data/sites.json:9245]:
```suggestion

    {
        "name": "McDonald's",
```

**Review by tupaschoal — APPROVED** [review]

**Pull request merged**